### PR TITLE
Do not use bias when batchnorm is used.

### PR DESCRIPTION
**Pull request by emailweixu**
It's a waste.

### CODE REVIEW

File: alf/layers.py [hunk 1]
```diff
@@ -700,7 +704,7 @@ def __init__(self,
             activation (torch.nn.functional):
             strides (int or tuple):
             padding (int or tuple):
-            use_bias (bool):
+            use_bias (bool): If None, will use ``not use_bn``
```
Comment (Haichao-Zhang) [inline alf/layers.py]: (bool) -> (bool|None)

File: alf/layers.py [hunk 2]
```diff
@@ -777,7 +783,7 @@ def __init__(self,
...
```
Comment (Haichao-Zhang) [inline alf/layers.py]: (bool) -> (bool|None)

File: alf/layers.py [hunk 3]
```diff
@@ -551,7 +553,7 @@ def __init__(self,
             activation (torch.nn.functional):
             strides (int or tuple):
             padding (int or tuple):
-            use_bias (bool): whether use bias
+            use_bias (bool): whether use bias. If None, will use ``not use_bn``
```
Comment (Haichao-Zhang) [inline alf/layers.py]: (bool) -> (bool|None)

File: alf/layers.py [hunk 4]
```diff
@@ -472,7 +472,7 @@ def __init__(self,
             activation (torch.nn.functional):
             strides (int or tuple):
             padding (int or tuple):
-            use_bias (bool): whether use bias
+            use_bias (bool|None): whether use bias. If None, will use ``not use_bn``
```
Comment (Haichao-Zhang) [inline alf/layers.py:475]: Other places can be updated like this one for the type of ``use_bias``

**Review by Haichao-Zhang — APPROVED** [review]
